app/routes/project/tasks/task.js

```diff
@@ -2,40 +2,23 @@ import Ember from 'ember';
 
 const {
   Route,
-  RSVP,
-  inject: { service },
-  isPresent
+  inject: { service }
 } = Ember;
 
 export default Route.extend({
   currentUser: service(),
 
   model(params) {
     let projectId = this.modelFor('project').id;
-    let queryParams = {
-      projectId,
-      number: params.number
-    };
+    let { number } = params;
 
-    let userId = this.get('currentUser.user.id');
-
-    return RSVP.hash({
-      task: this.store.queryRecord('task', queryParams),
-      user: isPresent(userId) ? this.store.find('user', userId) : null
-    }).then((result) => {
-      return RSVP.hash({
-        task: result.task,
-        comment: this.store.createRecord('comment', { task: result.task, user: result.user }),
-        comments: this.store.query('comment', { taskId: result.task.id })
-      });
-    });
+    return this.store.queryRecord('task', { projectId, number });
   },
 
-  setupController(controller, models) {
-    this._super(controller, models);
-    controller.set('task', models.task);
-    controller.set('newComment', models.comment);
-    controller.set('comments', models.comments);
+  setupController(controller, task) {
+    let user = this.get('currentUser.user');
```

Comment:
This is out of the scope of this PR, but I never like fetching properties directly from Services. It would be nice to have a `getUser` method:
`this.get('currentUser').getUser();`